client/my-sites/checkout/composite-checkout/components/form-field-annotation.tsx

```diff
@@ -1,6 +1,42 @@
 import styled from '@emotion/styled';
 import { FunctionComponent } from 'react';
```

Comment:
While you're touching this file, it would be nice to convert this to `import type`, too.

Reply:
Fixed!